src/components/CardLog/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Column from '../Column'
+import Row from '../Row'
+import Image from '../Image'
+import './index.css'
+
+const CardLog = props => {
+  return (
+    <div className='CardLog__container'>
+      <Row>
+        {props.cards
+          .filter((_, index) => index < 6)
+          .map((i, index) => {
+            return (
+              <Column key={i.id + ' #' + index} width={25}>
+                <Image
+                  className={
+                    'CardLog__image' +
+                    (index === 0 ? ' CardLog__latest_image' : '')
```

Comment:
```suggestion
                    (index === 0 ? ' DryRunnerCardLog__image--latest' : '')
```

For the record, this is called the BEM convention: Block (here `DryRunnerCardLog`, the component), Element (`image`), Modifier (`latest`).